OutOfSchool/OutOfSchool.DataAccess/Repository/ChangesLogRepository.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+using OutOfSchool.Common.Extensions;
+using OutOfSchool.Services.Extensions;
+using OutOfSchool.Services.Models;
+
+namespace OutOfSchool.Services.Repository
+{
+    public class ChangesLogRepository : EntityRepository<ChangesLog>, IChangesLogRepository
+    {
+        public ChangesLogRepository(OutOfSchoolDbContext dbContext)
+            : base(dbContext)
+        {
+        }
+
+        public ICollection<ChangesLog> AddChangesLogToDbContext<TEntity>(
```

Comment:
You fixed but it can be rewritten in short form as (for future):
```
_ = trackedProperties ?? throw new ArgumentNullException(nameof(trackedProperties));
_ = valueProjector?? throw new ArgumentNullException(nameof(valueProjector));
```
Can be marked as resolved after reading this comment.